### PR TITLE
TIEGCM comparison additions

### PR DESCRIPTION
**Pull request by rstoneback**
This pull includes a bugfix for the file format keyword in tiegcm. It also includes a first pass at fast interpolation for TIEGCM values along the satellite track. The method is sound but still needs to be updated to account for the log scaling of some TIEGCM parameters.

Creating this pull now before I forget about the changes II made a few months ago.

### CODE REVIEW

**Comment by jklenzing** [comment]
Adding latest develop commits to sort out recent testing bugs.

**Comment by rstoneback** [comment]
Thanks Jeff!

On Jun 4, 2019, at 9:22 AM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


Adding latest develop commits to sort out recent testing bugs.

File: pysat/model_utils.py [hunk 1]
```diff
@@ -16,6 +16,31 @@
 import numpy as np
 import pandas as pds
 
+def satellite_view_through_model(sat, tie, scoords, tlabels):
+    """Interpolates model values onto satellite orbital path.
+    
+    """
```
Comment (jklenzing) [inline pysat/model_utils.py]: needs proper docstring #TheEyeOfBurrell

**Comment by jklenzing** [comment]
Added a tentative docstring based on my reading of the code.  Also generalized inputs in case a string rather than a list of strings is input for either scoords or tlabels.  @rstoneback, am I interpreting this right?

**Comment by rstoneback** [comment]
Thanks for iterating on this Jeff. The module itself needs a bit of work. Parts of the data are linear, others are log. I need to work on ensuring consistency.

Already in Santa Fe?

On Jun 16, 2019, at 3:25 PM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


@jklenzing commented on this pull request.

Tiegcm tag labels refer to IVM

**Comment by jklenzing** [comment]
DFW airport. Was talking with Asher about TIEGCM capabilities, which is how I noticed this.

**Comment by rstoneback** [comment]
Heh. Sounds like everyone is at the same gate. I should mosey back that way.

On Jun 16, 2019, at 3:38 PM, Jeff Klenzing <notifications@github.com<mailto:notifications@github.com>> wrote:


DFW airport. Was talking with Asher about TIEGCM capabilities, which is how I noticed this.

**Comment by jklenzing** [comment]
Just switched to E35.

**Pull request merged**